### PR TITLE
Fix issue #850: lesson navigation

### PR DESCRIPTION
**Pull request by leosoaivan**

### CODE REVIEW

**Comment by KevinMulhern** [comment]
👍 LGTM
I have it on staging and it works great @leosoaivan ⭐️ 

@arun1595 should probably have a look over it before we ship it.

File: app/assets/javascripts/lessons.js [hunk 1]
```diff
@@ -67,14 +67,20 @@ function addActiveClass() {
...
-  var lessonNavigationElement = document.querySelector('.lesson-navigation');
-
-  lessonNavigationElement.innerHTML = lessonNavigationHTML;
-  addActiveClass();
-  Stickyfill.add(lessonNavigationElement);
+  if (commonHeadings.length < 2) {
+    var navigationColumn = document.querySelector('.lesson .col-lg-3');
+    var lessonColumn = document.querySelector('.lesson .row');
+
+    navigationColumn.style.display = 'none';
```
Comment (arun1595) [inline app/assets/javascripts/lessons.js]: How about using `navigationColumn.classList.add('hidden')` to stay consistent with the next line?

You'll have to define the CSS for `hidden` class or use [BS4's utility class](https://getbootstrap.com/docs/4.0/utilities/display/#hiding-elements) to hide elements.

Comment (leosoaivan) [reply app/assets/javascripts/lessons.js]: Sounds good, i'll take a look.

**Comment by arun1595** [comment]
@leosoaivan Thanks!

**Pull request merged**